[2022-12-09T00:02:52.434+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: back_in_avro.First_task_backup manual__2022-12-09T00:02:50.665861+00:00 [queued]>
[2022-12-09T00:02:52.458+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: back_in_avro.First_task_backup manual__2022-12-09T00:02:50.665861+00:00 [queued]>
[2022-12-09T00:02:52.461+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-09T00:02:52.463+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 6
[2022-12-09T00:02:52.465+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-09T00:02:52.496+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): First_task_backup> on 2022-12-09 00:02:50.665861+00:00
[2022-12-09T00:02:52.509+0000] {standard_task_runner.py:55} INFO - Started process 8678 to run task
[2022-12-09T00:02:52.518+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'back_in_avro', 'First_task_backup', 'manual__2022-12-09T00:02:50.665861+00:00', '--job-id', '40', '--raw', '--subdir', 'DAGS_FOLDER/dag_back_in_avro.py', '--cfg-path', '/tmp/tmpsacpjtqf']
[2022-12-09T00:02:52.523+0000] {standard_task_runner.py:83} INFO - Job 40: Subtask First_task_backup
[2022-12-09T00:02:52.638+0000] {task_command.py:389} INFO - Running <TaskInstance: back_in_avro.First_task_backup manual__2022-12-09T00:02:50.665861+00:00 [running]> on host c4fdd5f7672d
[2022-12-09T00:02:52.757+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Martin_Jurado
AIRFLOW_CTX_DAG_ID=back_in_avro
AIRFLOW_CTX_TASK_ID=First_task_backup
AIRFLOW_CTX_EXECUTION_DATE=2022-12-09T00:02:50.665861+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-09T00:02:50.665861+00:00
[2022-12-09T00:02:52.800+0000] {logging_mixin.py:137} INFO - There is information in the database
[2022-12-09T00:02:52.857+0000] {logging_mixin.py:137} INFO - Verificar archivos 
[2022-12-09T00:02:52.905+0000] {python.py:177} INFO - Done. Returned value was: None
[2022-12-09T00:02:52.919+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=back_in_avro, task_id=First_task_backup, execution_date=20221209T000250, start_date=20221209T000252, end_date=20221209T000252
[2022-12-09T00:02:52.974+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-12-09T00:02:53.009+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
